ts/parts/Legend.ts

```diff
@@ -1622,8 +1623,15 @@ Highcharts.Legend.prototype = {
 
             this.currentPage = currentPage;
             this.positionCheckboxes();
-        }
 
+            // Fire event after scroll animation is complete
+            const animOptions = H.animObject(
+                pick(animation, chart.renderer.globalAnimation, true)
+            );
+            setTimeout((): void => {
```

Comment:
Use `syncTimeout` so that it will run sync when duration is 0.

Reply:
Fixed in separate PR